### PR TITLE
Using the docker image

### PR DESCRIPTION
**Pull request by guillaumebriday**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ version: "3"
 
 services:
   excalidraw:
-    build: .
-    container_name: excalidraw
+    image: getexcalidraw/excalidraw:latest
```
Comment (kbariotis) [inline docker-compose.yml:5]: @guillaumebriday I'm not sure about this, the purpose of the `docker-compose` file is for local development. I don't see a reason to use the remote image for this.

**Comment by kbariotis** [comment]
Thanks for this @guillaumebriday. Unfortunately, we will have to wait a bit as we are waiting for Docker to release us the `excalidraw` organization so we don't use the `getexcalidraw` one. I have already raised a support ticket and waiting...

File: docker-compose.yml (hunk 1 above)
Comment (guillaumebriday) [reply docker-compose.yml:5]: I will change it

**Comment by guillaumebriday** [comment]
Hey

Any news from Docker?

Thanks

**Comment by kbariotis** [comment]
Nop unfortunately @guillaumebriday, I've sent them a message in every possible way and they dont seem to respond to any of them.

**Comment by kbariotis** [comment]
Closing this in favor of https://github.com/excalidraw/excalidraw/pull/1771

**Pull request closed without merging**